relayer/middleware/source-tx.middleware.ts

```diff
@@ -57,13 +69,23 @@ export function sourceTx(
   optsWithoutDefaults?: SourceTxOpts,
 ): Middleware<SourceTxContext> {
   let opts: SourceTxOpts;
+  let wormscan: WormscanClient;
   const alreadyFetchedHashes = new LRUCache({ max: 1_000 });
 
   return async (ctx, next) => {
     if (!opts) {
       // initialize options now that we know the environment from context
       opts = Object.assign({}, defaultOptsByEnv[ctx.env], optsWithoutDefaults);
     }
+    if (!wormscan) {
+      wormscan = new WormscanClient(new URL(opts.wormscanEndpoint), {
```

Comment:
We're trying to deprecate the name `wormscan` (hence the URL change), could we change all references to `wormscan` to `wormholescan` instead?

Reply:
makes sense

Reply:
changed